docs/plugins/graphback-datasync.md

```diff
@@ -0,0 +1,112 @@
+---
+id: dataSync-plugin
+title: Graphback Data Synchronization plugin
+sidebar_label: Data Synchronization plugin
+---
+
+# Graphback Data Synchronization plugin
+
+The Graphback Data Synchronization package consisting of the Data Synchronization Schema plugin, provides out of the box Data Synchronization strategies for GraphQL clients with offline functionality e.g. [Offix](https://offix.dev).
+
+# Installation
+
+The Graphback Data Synchronization package provides schema plugins as well as data sources for all supported data synchronization patterns.
+
+```bash
+npm install @graphback/datasync
+```
+
+# Usage
+
+There are two steps to implementing a data synchronization strategy:
+
+- Add metadata to schema
+- Use the relevant plugin and data source
+
+Currently the supported strategies are:
+
+- Soft deletes with delta queries for MongoDB
+
+## Soft Deletes with delta queries
+
+- ### Sprinkle metadata on your schema
+
+Add the `versioned` and `delta` markers to your model(s) in your GraphQL SDL:
```

Comment:
It will be simpler but they serve different purposes like we discussed
As an aside, would it be more helpful if we throw an error for example when somebody uses delta but without versioned?

Reply:
Yep. If delta is there we can have some validation